src/main/java/Decide.java

```diff
@@ -44,6 +44,37 @@ public static boolean LIC0(Point[] pts, Parameters params) {
         return false;
     }
 
+    public static boolean LIC8(Point[] pts, Parameters params) {
+        int a_pts = params.a_pts;
+        int b_pts = params.b_pts;
+        double radius1 = params.radius1;
+
+        if(a_pts < 1 && b_pts < 1) {
+            throw new IllegalArgumentException("A_PTS and B_PTS must be greater or equal to 1");
+        }
```

Comment:
Is it necessary to check if both is unfulfilled when they are checked individually?

Reply:
So I'll change